### PR TITLE
Phone verify

### PR DESCRIPTION
**Pull request by aviars**

### CODE REVIEW

File: apps/accounts/models.py [hunk 1]
```diff
@@ -401,15 +401,14 @@ def save(self, commit=True, **kwargs):
             self.code = str(random.randint(1000, 9999))
             up = UserProfile.objects.get(user=self.user)
             if self.mode == "SMS" and \
-               up.mobile_phone_number and \
-               settings.SEND_SMS:
+               up.mobile_phone_number:
                 # Send SMS to up.mobile_phone_number
                 sns = boto3.client(
                     'sns',
                     aws_access_key_id=settings.AWS_ACCESS_KEY_ID,
```
Comment (whytheplatypus) [inline apps/accounts/models.py]: Should just let the library get credentials from the system. 

https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html#guide-configuration

File: apps/accounts/models.py [hunk 2]
```diff
@@ -434,6 +433,51 @@ def save(self, commit=True, **kwargs):
...
+            now = pytz.utc.localize(datetime.utcnow())
+            expires = now + timedelta(days=1)
+            self.expires = expires
+            self.code = str(random.randint(1000, 9999))
+            up = UserProfile.objects.get(user=self.user)
+            if up.mobile_phone_number:
+                # Send SMS to up.mobile_phone_number
+                sns = boto3.client(
+                    'sns',
+                    aws_access_key_id=settings.AWS_ACCESS_KEY_ID,
```
Comment (whytheplatypus) [inline apps/accounts/models.py]: Let the library handle getting this from the system

File: apps/accounts/models.py [hunk 3]
```diff
@@ -434,6 +433,51 @@ def save(self, commit=True, **kwargs):
...
+                    aws_secret_access_key=settings.AWS_SECRET_ACCESS_KEY,
+                    region_name='us-east-1')
+                number = "%s" % (up.mobile_phone_number)
+                sns.publish(
+                    PhoneNumber=number,
+                    Message="Your code is : %s" % (self.code),
+                    MessageAttributes={
+                        'AWS.SNS.SMS.SenderID': {
+                            'DataType': 'String',
+                            'StringValue': 'MySenderID'
```
Comment (whytheplatypus) [inline apps/accounts/models.py]: does this need to be configurable?

Comment (aviars) [reply apps/accounts/models.py]: I don't think so...this is from an example.  I've never needed to deviate from this.

File: apps/accounts/models.py (hunk 1 above)
Comment (aviars) [reply apps/accounts/models.py]: @whytheplatypus yep.  Fixed.

File: apps/accounts/models.py (hunk 2 above)
Comment (aviars) [reply apps/accounts/models.py]: yes. Fixed.

**Pull request merged**